extractor/filesystem/sbom/spdx/spdx.go

```diff
@@ -116,7 +116,7 @@ func (e Extractor) convertSpdxDocToInventory(spdxDoc *spdx.Document, path string
 				packageURL, err := purl.FromString(extRef.Locator)
 				inv.Name = packageURL.Name
 				if err != nil {
-					log.Warnf("Invalid PURL for package: %q", extRef.Locator)
+					log.Warnf("Invalid PURL %q for package: %q", extRef.Locator, spdxPkg.PackageName)
```

Comment:
Ah wait this is not the PURL - the args should be packageURL, extRef.Locator

Reply:
Hmm I don't think so? When err is not nil, packageURL is likely to be empty. The (invalid) purl that we are converting from on line 116 is extRef.Locator no?

Reply:
ah you're right. I got confused by the fact that the second %q now prints the package instead of the PURL.